docs/events/list.md

```diff
@@ -1,143 +1,51 @@
 # Upcoming Events
 This list showcases upcoming UK student-run hackathons and hackathon-associated events.
 
-Want to add something to this list? [Fork and pull request](https://github.com/Hackathons-UK/wiki/edit/master/docs/index.md) to add your event!
+Want to add something to this list? [Fork and pull request](https://github.com/Hackathons-UK/wiki/edit/master/mkdocs.yml) to add your event! You can find the hackathons inside <code>hackathon_seasons:</code>
```

Comment:
can we use markdown code syntax (backticks) rather than html `code` tags?

Reply:
Sounds good, will sort!